Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+<html>
```

Comment:
- This document should be name index.html so it's the default page that loads when visiting this folder

Read: https://teamtreehouse.com/community/why-is-it-important-that-we-name-the-main-file-indexhtml